app/routes/events/components/JoinEventFormCountdownProvider.js

```diff
@@ -30,7 +30,7 @@ const COUNTDOWN_INTERVAL = 1000;
 // Must be sorted lo->hi
 const TICK_ACTIONS: Array<[number, Action]> = [
   [0, 'REGISTRATION_AVAILABLE'],
-  [3 * 60 * 1000, '3_MINUTE_LEFT'],
+  [3 * 60 * 1000, '90_SECONDS_LEFT'],
```

Comment:
Shouldn't the value also be changed to `90 * 1000` or `1.5 * 60 * 1000`?

Reply:
Oops, yes.